### DISCUSSION TITLE
Suggestion: Organize forms into "folders"

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-freeform#480] [Feature Requests]
Filing 3 separate suggestions following feedback from a client. We have a large multi-site Craft install with 8 sites. As the website continues to grow, our client is noticing some growing pains related to Freeform.

Suggestion 1: Could we add some kind of organization to the dashboard / list view? Currently we have everyone's forms all bundled in together, and over time - some of those forms will stop being used. It's currently impossible to tell which are active, who they belong to and how to find certain forms.

![image](https://user-images.githubusercontent.com/1250017/196878775-75f9b56b-86e4-4fa6-b842-6956c88f3af7.png)

**Comment by kjmartens** [comment]
Thanks for your feedback, @darylknight. The concept of folders/groups is something we're considering. 🙂

**Comment by caseyalee** [comment]
Yes, some sites have tons of forms. It would be great to have them navigable by group - similar to entry types.
Permissions per group would also be very nice to see in a future release.

**Comment by kjmartens** [comment]
@darylknight and @caseyalee,

I wanted to mention that the new Site Filtering feature (v5.2+) allows you to optionally restrict forms per Craft Site(s), so if you're using the multi-site feature, this will likely help significantly.

Additionally, we're working on a way to "archive" inactive forms from the main form dashboard page in Freeform 5.5.

We still plan to implement form groups/types in the future. 🙂

**Reply by darylknight** [reply]
This will be handy, thank you

**Comment by kjmartens** [comment]
Just an FYI that we have since released an Archive feature and are about to release a new Form Groups feature (where form cards can be organized by groups), and then a List view in another week or so. 🙂
